### PR TITLE
Pass more keyboard info to widgets

### PR DESCRIPTION
**Pull request by sapir**

### CODE REVIEW

File: crates/yakui-core/src/input/mod.rs [hunk 1]
```diff
@@ -5,3 +5,6 @@ mod input_state;
 
 pub use self::button::*;
 pub use self::input_state::*;
+
+pub use keyboard_types;
```
Comment (LPGhatguy) [inline crates/yakui-core/src/input/mod.rs:9]: Should we expose this whole crate or just the parts we use?

**Review by LPGhatguy — COMMENTED** [review]
This looks great! One comment I'm not sure about, but the rest looks A+.

File: crates/yakui-core/src/input/mod.rs (hunk 1 above)
Comment (sapir) [reply crates/yakui-core/src/input/mod.rs:9]: Probably the latter

Comment (sapir) [reply crates/yakui-core/src/input/mod.rs]: I've force-pushed a version that removes this line

**Review by LPGhatguy — APPROVED** [review]

**Pull request merged**